src/lib/Tweenable.js

```diff
@@ -0,0 +1,68 @@
+class Tweenable {
+    /**
+     * @param {number} [targetValue=0]
+     * @param {number} [startValue=targetValue]
+     * @param {number} [tweenTime=0]
+     * @param {number} [startTime=Date.now()]
+     * @param {(function(number): number)} [easing=Tweenable.Easing.EASE_IN_OUT_CUBIC]
```

Comment:
_All_ arguments are optional?

Also, what is `tweenTime`? Can that get an explaining doc comment? Is that the _duration_?

Reply:
Yes, it's the duration. Renamed it to `tweenDuration`.